src/main/kotlin/org/vaccineimpact/api/models/BurdenEstimate.kt

```diff
@@ -1,8 +1,11 @@
 package org.vaccineimpact.api.models
 
 import org.vaccineimpact.api.models.helpers.FlexibleColumns
+import org.vaccineimpact.api.models.helpers.FlexibleProperty
 import java.math.BigDecimal
 
+interface BurdenEstimateRow
```

Comment:
This is so we can have a single controller method that returns either central or stochastic data depending on a query parameter. Following the pattern used with coverage (wide/long).